src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/CompilationInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.CodeAnalysis.Razor.Workspaces;
+
+/// <summary>
+/// Information that we need from a Roslyn compilation
+/// </summary>
+[DataContract]
+internal record CompilationInfo(
```

Comment:
Any reason for this not to be a readonly record struct?

Reply:
Nope, good catch